bindings/node/.evergreen/setup_environment.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# set -o xtrace   # Write all commands first to stderr
```

Comment:
Up to you if you wanna change this. Though verbose, sometimes this helps debugging failures in evergreen since all executed commands are logged. If it there are commands that are super verbose, you could disable that around those commands with `set +o xtrace`.

Reply:
My concern with `set -o xtrace` was that we were using a secret token for the github push, and I did not want to risk that getting exposed. I might be a little overly paranoid on that front though, and I guess I could just border the `prebuild` push with `set +o xtrace`